core-rust-bridge/src/main/java/com/radixdlt/rev2/ScryptoConstants.java

```diff
@@ -79,10 +79,8 @@ public final class ScryptoConstants {
           .build(new TypeToken<Natives.Call1<Tuple.Tuple0, ComponentAddress>>() {})
           .call(Tuple.Tuple0.of());
 
-  public static final PackageAddress ACCOUNT_PACKAGE_ADDRESS =
-      Natives.builder(ScryptoConstants::getAccountPackageAddress)
-          .build(new TypeToken<Natives.Call1<Tuple.Tuple0, PackageAddress>>() {})
-          .call(Tuple.Tuple0.of());
+  // TODO(code review): how to replace this no-longer-existing static address?
+  public static final PackageAddress ACCOUNT_PACKAGE_ADDRESS = new PackageAddress(new byte[27]);
```

Comment:
There is no address currently but there will be one in a soon to come PR on the scrypto side. For now, this can be removed.